src/ophys_etl/modules/suite2p_wrapper/schemas.py

```diff
@@ -86,6 +86,55 @@ class Suite2PWrapperSchema(argschema.ArgSchema):
                     "of computing one internally. To use automated reference "
                     "image generation in ophys_etl, set this value to True and"
                     "refImg to an empty list or array (Default).")
+    do_optimize_motion_params = argschema.fields.Bool(
```

Comment:
Sorry. I should have noticed this before.

I feel like the parameters related to optimization (`do_optimize_motion_params`, `smooth_sigma_min/max` etc.) should all go in the schema for `suite2p_registration` rather than `suite2p_wrapper`. It would be nice, as much as possible, to keep `suite2p_wrapper` an in-the-raw call to Suite2P. This search through parameter space is our own functionality (indeed, the optimization code lives in suite2p_registration, rather than suite2p_wrapper).

This will force you to break out `smooth_sigma_min/max` from the `suite2p_args` dict in `optimize_motion_parameters`, but I think that is okay. Those aren't Suite2P parameters. They are our parameters.

Reply:
Sounds good. I'll get on it.

Reply:
Okay, here's what I ended up doing which also makes your comments above satisfied.

In moving the parameters out of suit2p_wrapper I also moved them out of the dict input to `optimize_motion_parameters`. They are now just arrays that go in, simplify the dictionaries and making them just suite2p_args dicts without the extra values.